search-sentiments-map/public/style.css

```diff
@@ -129,7 +143,8 @@ input:checked + .slider:before {
 
 #right-div {
   background-color: rgba(255, 255, 255, 0.5);
-  overflow: hidden;
+  height: 70%;

```

Comment:
I personally prefer the height to wrap the content (as in overflow: hidden). If you see the 70% height on a monitor there would be a lot of white space below

Reply:
Okay, but wrapping the overflow means that on small screens the user can still view all of the content. I feel like extra whitespace is a smaller problem.

Basically in overflow hidden, on smaller laptop screen, the content cuts off complete. I think extra white space on large monitors is less of a problem than content cutting off on laptops. I am attaching an image of what the cut off looks like vs the scrollable content.
![Screenshot 2020-08-05 at 9 52 52 AM](https://user-images.githubusercontent.com/33767556/89421063-913f3800-d701-11ea-8497-e5af83876389.png)
![Screenshot 2020-08-05 at 9 51 51 AM](https://user-images.githubusercontent.com/33767556/89421064-913f3800-d701-11ea-882e-0a65027b00db.png)

Reply:
Is the first the content cut that you talked about? I think I put a bottom padding to make it look less squeezed. But yeah if the screen is too small to display the entire panel then maybe we should go with the scroll

Reply:
Yeah if the screen is too small(like on a laptop), the content just cuts off.